server/routes/api/landing_page.py

```diff
@@ -41,20 +41,29 @@
 OVERVIEW = 'Overview'
 
 
-def get_landing_page_data(dcid, new_stat_vars):
-    return get_landing_page_data_helper(dcid, '^'.join(new_stat_vars))
+def get_landing_page_data(dcid, new_stat_vars, category,
+                          new_landing_page_cache_flag):
+    return get_landing_page_data_helper(dcid, '^'.join(new_stat_vars), category,
+                                        new_landing_page_cache_flag)
 
 
 @cache.memoize(timeout=3600 * 24)  # Cache for one day.
-def get_landing_page_data_helper(dcid, stat_vars_string):
+def get_landing_page_data_helper(dcid, stat_vars_string, category,
+                                 new_landing_page_cache_flag):
     data = {'place': dcid}
+    req_url = '/v1/internal/page/place/' + dcid
+    req_url += '?category={}&new_stat_vars={}'.format(
+        category, stat_vars_string.split(','))
```

Comment:
Can you move line 53 to 58 to the corresponding if, else block

Reply:
Done